buildenv/travis/build-on-travis.sh

```diff
@@ -71,11 +76,7 @@ if test "x$RUN_LINT" = "xyes"; then
   make -j $MAKE_JOBS -f linter.mk
 fi
 
-if test "x$RUN_BUILD" = "xyes"; then
-  if ! wget https://ci.eclipse.org/openj9/userContent/freemarker-2.3.8.jar -O freemarker.jar ; then
-    wget https://sourceforge.net/projects/freemarker/files/freemarker/2.3.8/freemarker-2.3.8.tar.gz/download -O freemarker.tgz
-    tar -xzf freemarker.tgz freemarker-2.3.8/lib/freemarker.jar --strip=2
-  fi
+if test "x$RUN_BUILD" = "xyes" ; then
```

Comment:
Has cmake removed the freemarker dependency?  I was under the impression it was still required beyond uma.

Reply:
I think UMA was the only use of freemarker. If you search openj9-openjdk-jdk11, you'll see that the only reference to `FREEMARKER_JAR` is in `OpenJ9.gmk` in a block that's not used with cmake.